gtfsmerger/__init__.py

```diff
@@ -2,6 +2,21 @@
 from gtfsmerger.gtfs import GTFS
 
 
+def post_process(func):
```

Comment:
I just think it might be cleaner by not chaining functions (or methods) inside merge method, making merge method procedurally go through other methods that do and converting would be another way to do it, but I don't think it's necessary to introduce this new hierarchy.